README.md

```diff
@@ -105,69 +133,81 @@ val csx = DataBag(Seq("Meijer", "Beckmann", "Wadler")) // computer scientists
 val fbx = DataBag(Seq("Meijer", "Pelé", "Meijer"))     // footballers
 
 // union (with bag semantics)
-csx plus fbx 
+csx union fbx
 // res: DataBag(Seq("Meijer", "Beckmann", "Pelé", "Wadler", "Meijer", "Meijer"))
 
-// difference (with bag semantics)
-fbx minus csx
-// res: DataBag(Seq("Pelé", "Meijer"))
-
 // duplicate elimination
-fbx distinct 
+fbx.distinct
 // res: DataBag(Seq("Meijer", "Pelé"))
 ```
 
 #### Nesting
 
-In addition to the collection manipulation primitives presented above, *Emma* offers a `groupBy` combinator, which (conceptually) groups bag elements by key and introduces a level of nesting. 
+In addition to the collection manipulation primitives presented above,
+*Emma* offers a `groupBy` combinator, which (conceptually) groups bag elements by key
+and introduces a level of nesting. 
 
-To compute the average size of email messages by sender, for example, one can simply write the following expression. 
+To compute the average size of email messages by sender, for example,
+one can simply write the following expression. 
 
 ```scala 
 for {
-  (sender, mails) <- emails.groupBy(_.from) 
+  Group(sender, mails) <- emails.groupBy(_.from) 
 } yield {
-  val sum = mails.map.(_.msg.length).sum()
-  val cnt = mails.count()
+  val sum = mails.map(_.msg.length).sum
+  val cnt = mails.size
   sum / cnt
 }
 ```
 
```

Comment:
We can add the following paragraph after the code snippet to highlight the benefits.

> If you express the above computation in a similar way in Flink's or Spark's APIs will result in dataflow graphs that do not scale well with the size of the input. Emma's takes a holistic optimization approach which allows it to transparently rewrite the program into a scalable Flink/Spark dataflows for you.